test/integration/concrete/lockup/withdraw/withdraw.t.sol

```diff
@@ -39,15 +39,68 @@ abstract contract Withdraw_Integration_Concrete_Test is Integration_Test, Withdr
         lockup.withdraw({ streamId: defaultStreamId, to: users.recipient, amount: withdrawAmount });
     }
 
-    function test_RevertWhen_FormerRecipient() external whenNotDelegateCalled givenNotNull givenStreamNotDepleted {
+    function test_RevertWhen_CallerSender()
+        external
+        whenNotDelegateCalled
+        givenNotNull
+        givenStreamNotDepleted
```

Comment:
How about we use `givenWithdrawalAddressNotRecipient` instead of `whenCallerUnauthorized`? `whenCallerUnauthorized` is really confusing. This test fails because caller is sender and withdrawal address is not recipient. But using `whenCallerUnauthorized` does not give that information away.

Reply:
Let's do this.